exercises/practice/two-fer/.meta/config.json

```diff
@@ -1,6 +1,16 @@
 {
   "blurb": "Create a sentence of the form \"One for X, one for me.\"",
   "authors": [],
```

Comment:
Do we want an author for that exercise?

Reply:
Well, if the author is empty the original author likely doesn't have a github username anymore, in which case we should leave it empty.

Reply:
Asking the silly question just in case: is existing/future tooling expecting empty `"authors"` ?

Reply:
Ideally, the authors should never be empty, but we can't do that for the older practices exercise if the user has since disappeared from github. Any new exercises _will_ have at least one author.

Reply:
Would that be the original pr?
https://github.com/exercism/csharp/pull/327

Reply:
Indeed it does! Looks like in this case the authorship information was somehow not found. Could you check the exercises without any authors and see if there is a corresponding PR?